src/main/scala/com/gu/octopusthrift/models/OctopusArticle.scala

```diff
@@ -0,0 +1,108 @@
+package com.gu.octopusthrift.models
+
+import java.util.concurrent.TimeUnit
+
+import com.gu.flexibleoctopus.model.thrift._
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.{ DateTime, DateTimeZone }
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+import scala.util.{ Success, Try }
+
+/**
+ * This model does not represent all fields sent by Octopus,
+ * only those that will be relevant in converting to our Thrift model
+ */
+case class OctopusArticle(
+  id: Int,
+  filename: String,
+  forPublication: String,
+  lawyered: String,
+  objectType: String,
+  objectNumber: Int,
+  lastModified: String,
```

Comment:
Is this the user or date? There's [an (unsaid) convention](https://github.com/guardian/flexible-model/blob/2b9509ad1150a7e56569ff8e7f3c2d4065628ad4/src/main/thrift/content.thrift#L1056-L1060) of calling this field `lastModifiedBy` when it's the user or `lastModified` of type `Long` or `Date` if it's the date.

Reply:
oh, unless this is a representation of the raw octopus json blob?

Reply:
This model represents a subset of the fields on the raw Octopus JSON, yes. This particular field represents a date and time, and is sent to us in a format like "202005271134".